docs/src/docs/arc42/cross-cutting/under-the-hood.adoc

```diff
@@ -46,54 +46,54 @@ When catching an exception, either log the exception and handle the problem or r
 ==== Write own base exceptions for (internal) interfaces
 By defining a common (checked) base exception for an interface, the caller is forced to handle potential errors, but can keep the logic simple. On the other hand, you still have the possibility to derive various, meaningful exceptions for different error cases, which can then be thrown via the API.
 
-Of course, when using only RuntimeExceptions, this is not necessary - but those can be overlooked quite easily, so be careful there.
+Of course, when using only RuntimeExceptions, this is not necessary - but these can be overlooked quite easily, so be careful there.
 
 ==== Central fallback exception handler
-There will always be some exception that cannot be handled inside of the code correctly - or it may just have been unforeseen. A central fallback exception handler is required so all problems are visible in the log and the API always returns meaningful responses. In some cases, this is as simple as a HTTP 500.
+There will always be some exception that cannot be handled inside the code correctly - or it may just have been unforeseen. A central fallback exception handler is required so all problems are visible in the log and the API always returns meaningful responses. In some cases, this is as simple as a HTTP 500.
```

Comment:
```suggestion
There will always be some exception that cannot be handled by the code correctly - or it may just have been unforeseen. A central fallback exception handler is required so all problems are visible in the log and the API always returns meaningful responses. In some cases, this is as simple as a HTTP 500.
```